src/utils/dig.js

```diff
@@ -7,38 +7,51 @@ const { createEmbed } = require('./embed');
 module.exports.handleDig = async ({ interaction, response, wait, domain, types, short }) => {
     // Make the DNS queries
     const results = [];
-    for (const type of types)
-        results.push({
-            type,
-            data: await performLookup(domain, type),
-        });
+
+    for (const type of types) results.push({
+        type,
+        // Following rule is disabled because the result of the statement after it is needed.
+        // eslint-disable-next-line no-await-in-loop
```

Comment:
I think we should just disable this rule at the config level, we're using `await` inside `for` in multiple places. Let's keep the code cleaner by just disabling it in the config instead of having to have disable comments in the code.